notes/BACKEND.md

```diff
@@ -1,3 +1,55 @@
 # Back-end Systems
 
-_Notes for this topic currently under development_
\ No newline at end of file
+This section adpated from Reference 1.
```

Comment:
sp: adapted

applies to multiple places in doc

Reply:
fixed.